### ISSUE TITLE
Draw a scaling Image Object on RenderTexture

### ISSUE THREAD

**Issue by kuoruan** [photonstorm/phaser#3454]
```
let image = new Phaser.GameObjects.Image(
  scene, 0, 0, 'base', 'frame'
)
image.setScale(0.5)

renderTexture.draw(image.texture, image.frame, 0, 0)
```
But the image is draw with the origin size.
How can I draw the image with scale.

**Comment by kuoruan** [comment]
I have tried, but failed. can you tell me the detail? @photonstorm

**Comment by kuoruan** [comment]
I set the RenderTexture as the image width and height, but it draw on RenderTexture with a wrong size.

**Comment by samid737** [comment]
@kuoruan You can adjust the currentMatrix property of the rendertexture, which transforms the (to be rendered) texture: 

http://jsfiddle.net/rgad0vb2/7/

It is not straightforward facing the API (yet), but with A transformation matrix you can do lots of things. 

In the example you are stretching/scaling in both horizontal and vertical direction by some (random) number _k_ :

![image](https://user-images.githubusercontent.com/30023855/38083821-0160d7e0-334b-11e8-9e04-a0deb449d63f.png)

**Comment by kuoruan** [comment]
@samid737 Thank you very much. now it's work.
I set the matrix to [1, 0, 0, 0.5, 0, 0], but don't know why the default [1, 0, 0, 1, 0, 0] will scale the image.

**Comment by samid737** [comment]
Yes, so the scaling can be done in x and y. If you set the matrix to [1, 0, 0, 0.5, 0 ,0] then k = 0.5 in y direction (which squeezes the image vertically). Then again, if both entries are one, k = 1, then the image will have its original scale. You can all sorts of crazy things with the transform (rotations, shearing, skewing) by adjusting the numbers inside currentMatrix. for example, shearing an image:

![image](https://user-images.githubusercontent.com/30023855/38085179-ebe20916-334f-11e8-923e-884400ddf0ac.png)

which becomes:

`rt.currentMatrix = [1, k , 0 ,1 , 0 ,0]` or `rt.currentMatrix = [1, 0 , k ,1 , 0 ,0]`

**Comment by photonstorm** [comment]
Here, you can use the methods exposed in the MatrixStack component:

```
function preload ()
{
    this.load.image('bunny', 'assets/sprites/bunny.png');
}

function create() {

    //  Create a Render Texture at 0x0 which is 800x600 in size
    var rt = this.add.renderTexture(0, 0, 800, 600);

    //  Get a texture to draw with
    var bunny = this.textures.getFrame('bunny');

    rt.rotate(0.9);
    rt.scale(0.5, 0.5);
    rt.translate(600, 100);

    rt.draw(bunny.texture, bunny, 0, 0);
}
```

**Comment by samid737** [comment]
@photonstorm, overlooked the API methods. I will probably re-adjust the [rendertexture examples PR](https://github.com/photonstorm/phaser3-examples/pull/90) too for readability. Was getting used to modding currentMatrix instead!

**Comment by photonstorm** [comment]
I think there are cases where changing the matrix directly would be handy. It's currently flagged as private in the docs, but I'm going to swap it to public.
